src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -33,9 +33,19 @@ export namespace ValidationSchemas {
                 passphrase: joi.string()
             })),
             trial_config: joi.object({
```

Comment:
it might be better to improve this validation, this mixes the validation of trial's local/remote mode and pai mode.